Fix github.com/hashicorp/go.net to build on arm64.

github.com/hashicorp/go.net is a fork of golang.org/x/net. Ideally
the upstream should stop using the fork.
https://github.com/hashicorp/mdns/issues/62

--- a/src/github.com/hashicorp/go.net/ipv4/syscall_unix.go	2019-01-07 17:30:32.581196443 +0900
+++ b/src/github.com/hashicorp/go.net/ipv4/syscall_unix.go	2019-01-08 12:57:18.722394723 +0900
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// +build darwin dragonfly freebsd linux,amd64 linux,arm netbsd openbsd
+// +build darwin dragonfly freebsd linux,amd64 linux,arm linux,arm64 netbsd openbsd
 
 package ipv4
 
--- a/src/github.com/hashicorp/go.net/ipv6/syscall_unix.go	2019-01-07 17:30:39.525112617 +0900
+++ b/src/github.com/hashicorp/go.net/ipv6/syscall_unix.go	2019-01-08 13:12:31.079355040 +0900
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// +build darwin dragonfly freebsd linux,amd64 linux,arm netbsd openbsd
+// +build darwin dragonfly freebsd linux,amd64 linux,arm linux,arm64 netbsd openbsd
 
 package ipv6
 
